pipe-cli/src/api/pipeline.py

```diff
@@ -259,6 +260,16 @@ def get_estimated_price(cls, pipeline_id, version, instance_type, instance_disk,
         response_data = api.call(api_url, json.dumps(data))
         return InstancePrice.load(response_data['payload'])
 
+    @classmethod
+    def get_default_run_parameters(cls):
+        api = cls.instance()
+        response_data = api.call('/run/defaultParameters', None)
+        result = []
+        if 'payload' in response_data:
+            for parameter_json in response_data['payload']:
+                result.append(PipelineRunParameterModel.load_from_default_system_parameter(parameter_json))
```

Comment:
It can be written just a little bit easier.

```suggestion
        for parameter_json in response_data.get('payload', []):
            result.append(PipelineRunParameterModel.load_from_default_system_parameter(parameter_json))
```

Reply:
Done